src/NIDAQ.jl

```diff
@@ -20,10 +20,10 @@ More examples are in tests/.
 """
 module NIDAQ
 
-import Base.write, Base.read, Base.start
+import Base.write, Base.read
 
 # tasks
-export stop, clear
+export start, stop, clear
```

Comment:
i'd suggest to continue to export `start` so than it does not have to be qualifed by `NIDAQ.`